From 620a4ca3ae788ca81ea1d52b9f43ac0cf1607203 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Mon, 1 Apr 2019 15:57:58 +0800
Subject: [PATCH 237/245] nxp-ls1043: remove no-map attribute for reserved
 memory

When performing kexec, qman/bman driver need access reserved memory to
determine device state in second kernel. So remove no-map attributes to
allow kernel access those area.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
index 7e0e334b6416..3bdd193dad8e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -113,21 +113,18 @@
 			compatible = "shared-dma-pool";
 			size = <0 0x1000000>;
 			alignment = <0 0x1000000>;
-			no-map;
 		};
 
 		qman_fqd: qman-fqd {
 			compatible = "shared-dma-pool";
-			size = <0 0x400000>;
-			alignment = <0 0x400000>;
-			no-map;
+			size = <0 0x800000>;
+			alignment = <0 0x800000>;
 		};
 
 		qman_pfdr: qman-pfdr {
 			compatible = "shared-dma-pool";
 			size = <0 0x2000000>;
 			alignment = <0 0x2000000>;
-			no-map;
 		};
 	};
 
-- 
2.17.1

